python/export_util.py

```diff
@@ -108,6 +113,274 @@ def get_graph(ctx: mgp.ProcCtx) -> List[Union[Node, Relationship]]:
     return nodes + relationships
 
 
+def get_graph_info_from_lists(node_list: list, relationship_list: list):
+    graph = list()
+    all_node_properties = list()
+    all_node_prop_set = set()
+    all_relationship_properties = list()
+    all_relationship_prop_set = set()
+
+    for node in node_list:
+        for prop in node.properties:
+            if prop not in all_node_prop_set:
+                all_node_properties.append(prop)
+                all_node_prop_set.add(prop)
+        graph.append(Node(node.id, node.labels, node.properties))
+    all_node_properties.sort()
+
+    for relationship in relationship_list:
+        for prop in relationship.properties:
+            if prop not in all_relationship_prop_set:
+                all_relationship_properties.append(prop)
+                all_relationship_prop_set.add(prop)
+
+        graph.append(
+            Relationship(
+                relationship.to_vertex.id,
+                relationship.id,
+                relationship.type.name,
+                relationship.properties,
+                relationship.from_vertex.id,
+            )
+        )
+    all_relationship_properties.sort()
+
+    return graph, all_node_properties, all_relationship_properties
+
+
+def csv_header(
+    node_properties: List[str], relationship_properties: List[str]
+) -> List[str]:
+    """
+    This function creates the header for csv file
+    """
+    header = ["_id", "_labels"]
+
+    for prop in node_properties:
+        header.append(prop)
+
+    header.extend(["_start", "_end", "_type"])
+
+    for prop in relationship_properties:
+        header.append(prop)
+
+    return [header]
+
+
+def process_properties(properties, prop, write_list):
+    if isinstance(properties[prop], (set, list, tuple, map)):
+        write_list.append(js.dumps(properties[prop]))
+    else:
+        if isinstance(properties[prop], timedelta):
+            write_list.append(convert_to_isoformat(properties[prop]))
+        else:
+            write_list.append(properties[prop])
+
+
+def csv_data_list(
+    graph: List[Union[Node, Relationship]],
+    node_properties: List[str],
+    relationship_properties: List[str],
+) -> list:
```

Comment:
Here instead of list, set it to List, because it is from Typing module, and also add what are you returning, List[X] where X is either str, int, ... can be also from Typing.Any